tests/pytest/snapshot_test.py

```diff
@@ -0,0 +1,25 @@
+from test_dragonfly import DRAGONFLY_PATH, dfly_args, df_server, client, connection
+
+import redis
+from pathlib import Path
+import time
+import pytest
+
+OUT_DIR = Path(DRAGONFLY_PATH).parent
+
+@pytest.mark.usefixtures("client")
```

Comment:
out of curiosity, does not pytest recognize fixtures automatically?

Reply:
It does. I wasn't sure for class based tests originally.